ui/lobby/src/ctrl.ts

```diff
@@ -74,9 +74,9 @@ export default class LobbyController {
 
     const locationHash = location.hash.replace('#', '');
     if (['ai', 'friend', 'hook'].includes(locationHash)) {
-      let friendUser: string;
       const forceOptions: ForceSetupOptions = {};
       const urlParams = new URLSearchParams(location.search);
+      const friendUser = urlParams.get('user') ?? undefined;
```

Comment:
why is adding `?? undefined` necessary here?

Reply:
`params.get(...)` returns `null` when the parameter is not found (i.e. `string | null`), but the `friendUser` argument of `openModal` a few lines below accepts `string | undefined`, so the `null` value has to be converted to `undefined`.